app/components/chart-published.tsx

```diff
@@ -142,13 +159,22 @@ export const ChartPublished = (props: ChartPublishedProps) => {
                 <Description text={state.layout.meta.description[locale]} />
               )}
             </Box>
-
-            <ChartPanelLayout
-              layoutType={state.layout.layout}
-              chartConfigs={state.chartConfigs}
-              renderChart={renderChart}
-            />
-          </>
+            <Flex
+              style={{
+                gap: 16,
+              }}
+              sx={{
+                flexDirection: "column",
+              }}
+            >
+              <ChartPanelLayout
+                layoutType={state.layout.layout}
+                chartConfigs={state.chartConfigs}
+                renderChart={renderChart}
+              />
+              <VisualizeLink />
```

Comment:
I think we should only add link when we are previewing a dashboard, see that it's already shown in footnotes in case we look at a single chart: https://github.com/visualize-admin/visualization-tool/blob/03f3eef58b288fe25dc8784b55dee54763550c1c/app/components/chart-published.tsx#L421

Reply:
Isn't state.layout.type === "dashboard" checking if it's dashboard?

Reply:
Ah, my bad, I didn't see that there's code below that renders non-dashboard charts 😅 Still, we could have a situation of a dashboard with one chart, e.g. when we edit a layout with two charts, and then remove one while in layouting step (see that in the screenshot below the chart already contains a link).

<img width="1253" alt="Screenshot 2024-09-13 at 14 20 06" src="https://github.com/user-attachments/assets/caf42bfd-d0e6-4e3a-8a0d-98f8ba381048">

So maybe in fact, there should be a safe-guard added to `showVisualizeLink={state.chartConfigs.length === 1}`, like:

```ts
showVisualizeLink={state.chartConfigs.length === 1 && state.layout.type !== "dashboard"}
```

so in such cases we only show the text link that you added in this PR, outside of the chart itself?

Reply:
At least in Kerstins design for dashboards it's not in the individual charts.